HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -329,7 +335,7 @@ def self.run_validator(hpxml_doc)
         '../../HVACDistribution[DistributionSystemType/AirDistribution | DistributionSystemType[Other="DSE"]]' => one_or_more, # See [HVACDistribution]
         'DistributionSystem' => one,
         'CoolingCapacity' => one, # Use -1 for autosizing
-        '[CompressorType="single stage" or CompressorType="two stage" or CompressorType="variable speed"]' => zero_or_one,
+        '[CompressorType="single stage" or CompressorType="two stage" or CompressorType="variable speed" or not(CompressorType)]' => one,
```

Comment:
Do you mind moving the not(Foo) to the front? I'm concerned that someone might not notice it tacked on the end. Comment applies to other parts of EPvalidator too.